openpype/hosts/houdini/plugins/load/load_ass.py

```diff
@@ -105,3 +67,23 @@ def remove(self, container):
 
         node = container["node"]
         node.destroy()
+
+    @staticmethod
+    def format_path(path, representation):
+        """Format file path correctly for single bgeo or bgeo sequence."""
+        if not os.path.exists(path):
+            raise RuntimeError("Path does not exist: %s" % path)
+
+        is_sequence = bool(representation["context"].get("frame"))
+        # The path is either a single file or sequence in a folder.
+        if not is_sequence:
+            filename = path
+        else:
+            filename = re.sub(r"(.*)\.(\d+)\.(ass.*)", "\\1.$F4.\\3", path)
+
+            filename = os.path.join(path, filename)
```

Comment:
Shouldn't it be like this?
```suggestion
            filename = os.path.join(os.path.dirname(path), filename)
```

Reply:
Or to completelly simplify the function just
```
if is_sequence:
    dirpath, filename = os.path.split(path)
    new_filename = re.sub(r"(.*)\.(\d+)\.(ass.*)", "\\1.$F4.\\3", filename)
    path = os.path.join(dirpath, new_filename )

return os.path.normpath(path).replace("\\", "/")
```